pyproject.toml

```diff
@@ -141,22 +140,24 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-aarch64-image = "manylinux2014"
 before-build = "bash {project}/tools/wheels/cibw_before_build_linux.sh {project}"
 
+[tool.cibuildwheel.linux.environment]
+# /project will be the $PWD equivalent inside the docker used to build the wheel
+PKG_CONFIG_PATH="/project/"
+
 [tool.cibuildwheel.macos]
-before-build = "bash {project}/tools/wheels/cibw_before_build_macos.sh {project}"
+before-build = "pip install ninja; bash {project}/tools/wheels/cibw_before_build_macos.sh {project}"
+
+[tool.cibuildwheel.macos.environment]
+# On macos-arm64, use Accelerate instead of setting this
+# CIBW_CONFIG_SETTINGS=\"setup-args=-Dblas=accelerate
+PKG_CONFIG_PATH="{project}"
 
 [tool.cibuildwheel.windows]
-before-build = "bash {project}/tools/wheels/cibw_before_build_win.sh {project}"
+before-build = "pip install ninja && bash {project}/tools/wheels/cibw_before_build_win.sh {project}"
 repair-wheel-command = "bash ./tools/wheels/repair_windows.sh {wheel} {dest_dir}"
 
-[[tool.cibuildwheel.overrides]]
-select = "*-win32"
-
-[[tool.cibuildwheel.overrides]]
-select = "*-win_amd64"
-# can use pkg-config detection for win_amd64 because the installed rtools
-# provide a working pkg-config.
-# An alternative is to set CMAKE_PREFIX_PATH="c:/opt/openblas/if_32/32"
-# Don't use double backslash for path separators, they don't get passed
-# to the build correctly
-# environment = { CMAKE_PREFIX_PATH="c:/opt/64" }
-environment = { PKG_CONFIG_PATH = "c:/opt/64/lib/pkgconfig" }
+[tool.cibuildwheel.windows.environment]
+# This does not work because pkg-config does not like backslashes,
+PKG_CONFIG_PATH="{project}"
+# do this instead (which will override this setting)
+# set CIBW_ENVIRONMENT_WINDOWS=PKG_CONFIG_PATH=PWD.replace('\\', '/')
```

Comment:
what is happening with this comment?

Reply:
You cannot set `PKG_CONFIG_PATH` to `"{project}"` on windows, the path native separators confuse `pkg-config`. So you have to set it via CIBW_ENVIRONMENT_WINDOWS instead. This is a comment for people wishing to build wheels that they must use that alternative. Should I remove it?

Reply:
Ah okay. I didn't really understand that from the comment - let's just clarify it instead.

To be honest I also don't understand the "this does not work" comment (which I now think is also aimed at folks wanting to build locally?), nor what `"{project}"` expands to and which tool does the expanding.

Reply:
Okay, let's deal with this loose end post-merge. Time to get this in, so we have it in the 1.14.x branch - that'll make maintenance easier.